python/paddle/tensor/linalg.py

```diff
@@ -3350,3 +3350,116 @@ def corrcoef(x, rowvar=True, name=None):
         c = paddle.clip(c, -1, 1)
 
     return c
+
+
+def cdist(
+    x, y, p=2.0, compute_mode="use_mm_for_euclid_dist_if_necessary", name=None
+):
+    r"""
+
+    Compute the p-norm distance between each pair of the two collections of inputs.
+
+    This function is equivalent to `scipy.spatial.distance.cdist(input,'minkowski', p=p)`
+        if :math:`p \in (0, \infty)`. When :math:`p = 0` it is equivalent to `scipy.spatial.distance.cdist(input, 'hamming') * M`.
+        When :math:`p = \infty`, the closest scipy function is `scipy.spatial.distance.cdist(xn, lambda x, y: np.abs(x - y).max())`.
+
+    Args:
+        x (Tensor): Tensor, shape is :math:`B \times P \times M`.
+        y (Tensor): Tensor, shape is :math:`B \times R \times M`.
+        p (float, optional): The value for the p-norm distance to calculate between each vector pair. Default: :math:`2.0`.
+        compute_mode (str, optional): the mode for compute distance.
+            - 'use_mm_for_euclid_dist_if_necessary', for p = 2.0 and (P > 25 or R > 25), it will use
+                matrix multiplication to calculate euclid distance if possible.
+            - 'use_mm_for_euclid_dist', for p = 2.0, it will use matrix multiplication to calculate euclid distance.
+            - 'donot_use_mm_for_euclid_dist', it will not use matrix multiplication to calculate euclid distance.
+            Default: 'use_mm_for_euclid_dist_if_necessary'.
+        name (str, optional): For details, please refer to :ref:`api_guide_Name`. Generally, no setting is required. Default: None.
+
+    Returns:
+        Tensor, the dtype is same as input tensor.
+
+        If x has shape :math:`B \times P \times M` and y has shape :math:`B \times R \times M` then
+            the output will have shape :math:`B \times P \times R`.
```

Comment:
格式错误
![image](https://github.com/PaddlePaddle/Paddle/assets/70642955/a12a68ca-4f68-4887-907d-3ab013532ae3)
```suggestion
        If x has shape :math:`B \times P \times M` and y has shape :math:`B \times R \times M` then
        the output will have shape :math:`B \times P \times R`.
```